tests/snapshot_tests/snapshot_apps/command_palette_key.py

```diff
@@ -0,0 +1,15 @@
+from textual.app import App, ComposeResult
+from textual.widgets import Footer
+
+
+class NewPaletteBindingApp(App):
+    COMMAND_PALETTE_BINDING = "ctrl+backslash"
+    COMMAND_PALETTE_DISPLAY = "ctrl+\\"
```

Comment:
Perhaps I've misunderstood this new `COMMAND_PALETTE_DISPLAY` variable , but why does the snapshot show `^/` rather than `ctrl+\`?

Reply:
There is some work in https://github.com/Textualize/textual/pull/4876 that will fix that.

Reply:
It might be worth adding that work in this PR if possible? This might help debugging later, as really the snapshot is not as expected.